corehq/apps/ota/views.py

```diff
@@ -184,7 +184,8 @@ def _add_case_property_queries(domain, case_type, search_es, criteria):
             case_property=key,
         )
         for removal_regex in remove_char_regexs:
-            value = re.sub(removal_regex.regex, '', value)
+            to_remove = re.escape(removal_regex.regex)
```

Comment:
hey, I think let's leave it as a regex for now. I've updated the app config for now, and will update the docs / do some validation when I get back on Tuesday.